processing-status-api-function-app/src/main/kotlin/gov/cdc/ocio/processingstatusapi/functions/reports/ServiceBusProcessor.kt

```diff
@@ -79,8 +79,8 @@ class ServiceBusProcessor(private val context: ExecutionContext) {
         logger.info("Creating report for uploadId = $uploadId with stageName = $stageName")
         ReportManager().createReportWithUploadId(
```

Comment:
Creating reports needs to support V1 and V2 metadata.  So, you'll need to update CreateReportSBMessage.kt model to include the destination_id and event_type.  If V1 is provided we will convert it to V2 names when writing them to cosmos.

Reply:
Corrected code to accept both V1 and V2 messages